scripts/toolboxSrc/pgEnsureExtensions.ts

```diff
@@ -0,0 +1,14 @@
+import {sql} from 'kysely'
+import getKysely from '../../packages/server/postgres/getKysely'
+
+export default async () => {
+  console.log('🔩 Postgres Extension Checks Started')
+  if (process.env.POSTGRES_USE_PGVECTOR) {
```

Comment:
you'll have to write `=== 'true'` since our dotenv parser does not parse the string `"true"` to the binary type `true`.

Reply:
Ugh – color me embarrassed. Coming right up sir